### PR TITLE
migrate to github actions

### LINKED ISSUE #36: migrate CI to GitHub actions

### PR DESCRIPTION
**Pull request by azjezz**
closes #36

### CODE REVIEW

**Comment by azjezz** [comment]
ready.

File: .github/workflows/coding-standards.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        uses: "actions/checkout@v2"
+
+      - name: "installing PHP"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          php-version: "7.4"
+          ini-values: memory_limit=-1
+          tools: composer:v2, cs2pr
+
+      - name: "installing php-cs-fixer"
```
Comment (dunglas) [inline .github/workflows/coding-standards.yml:22]: Can't you use the PHP CS Fixer provided by setup-php?

**Review by dunglas — APPROVED** [review]

File: .github/workflows/coding-standards.yml (hunk 1 above)
Comment (azjezz) [reply .github/workflows/coding-standards.yml:22]: ah! forgot about `tools`, will do 👍

Comment (azjezz) [reply .github/workflows/coding-standards.yml]: done

**Pull request merged**